snap/get_version

```diff
@@ -0,0 +1,114 @@
+#!/bin/dash
```

Comment:
As a general comment, since this script can only run in a snapcraft build environment, it should make sure it is actually running in a snapcraft environment.

Reply:
For debugging or adjusting, I still like to be able to run it. This is under `snap/` directory, and it’s using `SNAPCRAFT_PART_SRC` which needs to be exported with `set -u`. Do you think this is not enough?

Reply:
what about craftctl? Does this command exists outside of snapcraft?

Reply:
As answered on MM, the command will fail (which is generally a no-op). What‘s the issue with this. But again, the script will first fail if you don’t explicit set the SNAPCRAFT env variable. This is not that different from the install hooks/ in `snap/hooks` or in the debian context you have debian/rules that you shouldn’t run by hand. If the script was outside of the `snap/` directory, I would udnderstand, but here, I’m not understanding what‘s the actual concern?

Reply:
okay then